python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -529,15 +538,17 @@ def __init__(cls, name, bases, this_dict):
 
         # all Ganga classes must have (even empty) schema
         if cls._schema is None:
-            cls._schema = Schema.Schema(None, None)
+            cls._schema = Schema(None, None)
 
         this_schema = cls._schema
 
+        cls.__slots__ = ('__dict__', '_proxyObject')
```

Comment:
I'm following http://stackoverflow.com/a/28059785 and of the opinion there isn't a good reason not to use? I'll benchmark in a bit, putting in other changes to clean up this PR to be minimal. It's evolved a lot